Sources/SymbolKit/SymbolGraph/Misc/LineList.swift

```diff
@@ -92,7 +92,7 @@ extension SymbolGraph {
         /// The lines making up this line list.
         public var lines: [Line]
 
-        init(_ lines: [Line]) {
+        public init(_ lines: [Line]) {
```

Comment:
Why we need LineList public here? I did not see any usage in the #10 and https://github.com/apple/swift-docc/pull/61/

Reply:
This PR is also needed in https://github.com/apple/swift-package-manager/pull/3953.